src/pybind/mgr/volumes/module.py

```diff
@@ -147,6 +147,16 @@ def _cmd_fs_volume_create(self, inbuf, cmd):
         if r != 0:
             return r, outb, outs
 
+        # set higher recovery priority on metadata pool
+        r, outb, outs = self.mon_command({
+            'prefix': 'osd pool set',
+            'pool': mdp_name,
+            'var': "recovery_priority",
+            'val': "5",
+        })
+        if r != 0:
+            return r, outb, outs
```

Comment:
Why not set this when creating the file system?

Reply:
no good reason.. should we move the other adjustments being made here too?

Reply:
Sorry didn't get a github notification on this for some reason. Yes, I think so!